cmd/cnbBuild.go

```diff
@@ -498,14 +568,6 @@ func runCnbBuild(config *cnbBuildOptions, telemetryData *telemetry.CustomData, u
 		return errors.Wrap(err, "failed to generate CNB_REGISTRY_AUTH")
 	}
 
-	if dockerConfigFile != "" {
-		err = utils.FileRemove(dockerConfigFile)
```

Comment:
We cannot remove the `dockerConfig.json` anymore because for `multipleImages`, the next call might need it.

Reply:
Well, we could, if we would take the handling out of the single `runCnbBuild`, couldn't we?